tests/profiles/test_base_profile.py

```diff
@@ -47,6 +49,75 @@ def test_disable_event_tracking(disable_event_tracking: str):
     )
     profile_contents = yaml.safe_load(test_profile.get_profile_file_contents(profile_name="fake-profile-name"))
 
-    assert ("config" in profile_contents) == disable_event_tracking
```

Comment:
This line should still be kept here to assert that if only `disable_event_tracking` is used and is False that config isn't added to the profile contents.

Reply:
fix